### DISCUSSION TITLE
Join Our New Discord Community!

### DISCUSSION THREAD

**Discussion by kaplanelad** [loco-rs/loco#197] [Announcements]
We've just launched a new Discord community to enhance our interactions and provide a space for all of us to connect. 

https://discord.gg/fTvyBzwKS8

Feel free to drop by, say hello, and become a part of our growing community!

**Comment by sidoqi** [comment]
Hi @kaplanelad, the invitation link looks expired. Could you help to resend it? Thanks!

**Reply by kaplanelad** [reply]
@jurukode thanks for the notice.  the link updated
